en/docs/basics/02_tensor.md

```diff
@@ -0,0 +1,185 @@
+```
+Out:
+```text
+tensor([[1., 1., 1.],
+        [1., 1., 1.]], dtype=oneflow.float64)
+tensor([[0.6213, 0.6142, 0.1592],
+        [0.5539, 0.8453, 0.8576]], dtype=oneflow.float64)
+```
+
+
+### Created by an operator 
+
+There are also many operators available in OneFlow that you can use to create tensors. For example, [ones](https://oneflow.readthedocs.io/en/master/oneflow.html?highlight=oneflow.ones#oneflow.ones), [zeros](https://oneflow.readthedocs.io/en/master/oneflow.html?highlight=oneflow.zeros#oneflow.zeros) and [eye](https://oneflow.readthedocs.io/en/master/oneflow.html?highlight=oneflow.eye#oneflow.eye) create all 1’s tensor, all 0’s tensor, and the unit tensor, respectively.
+
+
+```python
+x5 = flow.ones(2, 3)
+x6 = flow.zeros(2, 3)
+x7 = flow.eye(3)
+print(x5)
+print(x6)
+print(x7)
+```
+
+Out:
+```text
+tensor([[1., 1., 1.],
+        [1., 1., 1.]], dtype=oneflow.float32)
+tensor([[0., 0., 0.],
+        [0., 0., 0.]], dtype=oneflow.float32)
+tensor([[1., 0., 0.],
+        [0., 1., 0.],
+        [0., 0., 1.]], dtype=oneflow.float32)
+```
+
+The [randn](https://oneflow.readthedocs.io/en/master/oneflow.html?highlight=oneflow.randn#oneflow.randn) method creates a randomized tensor:
+
+```python
+x8 = flow.randn(2,3)
+print(x8)
+```
+
+## The difference between `Tensor` and `tensor`
+There are two interfaces([oneflow.Tensor](https://oneflow.readthedocs.io/en/master/tensor.html?highlight=oneflow.Tensor#oneflow.Tensor) and [oneflow.tensor](https://oneflow.readthedocs.io/en/master/oneflow.html?highlight=oneflow.Tensor#oneflow.tensor) ) in OneFlow, both of which can be used to create tensors. What’s the difference?
+
+
+Briefly speaking, the data type of `oneflow.Tensor` is limited to FLOAT32 by default, while the data type of `oneflow.tensor` can be changed when the data is created. The following code illustrates the difference:
+
+```python
+print(flow.Tensor([1, 2, 3]))
+print(flow.tensor([1, 2, 3]))
+print(flow.tensor([1.0, 2.0, 3.0]))
+```
+
+Out:
+```text
+tensor([1., 2., 3.], dtype=oneflow.float32)
+tensor([1, 2, 3], dtype=oneflow.int64)
+tensor([1., 2., 3.], dtype=oneflow.float32)
+```
+
+Besides, `oneflow.Tensor` can be created without specifying specific data:
+
+```python
+x9 = flow.Tensor(2, 3)
+print(x9.shape)
+```
+
+Out:
+```text
+flow.Size([2, 3])
+```
+
+Therefore, use `oneflow.Tensor` to create a tensor if you do not want to specify the data type, otherwise, you should use `oneflow.tensor`.
+
+## Attributes of Tensor
+
+The `shape`, `dtype`, and `device` attributes of a tensor describe its shape, data type, and device type respectively.
+
+```python
+x9 = flow.randn(1,4)
+print(x9.shape)
+print(x9.dtype)
+print(x9.device)
+```
+
+Out:
+```text
+flow.Size([1, 4])
+oneflow.float32
+cpu:0
+```
+
+The output shows the shape, the data type, and the device (on CPU number 0, CPUs were numbered because OneFlow naturally supports distribution, see [Consistent Tensor](../parallelism/03_consistent_tensor.md)) of the tensor.
```

Comment:
```suggestion
The output shows the shape, the data type, and the device (on CPU number 0, CPUs were numbered because OneFlow naturally supports distribution, see [Consistent Tensor](../parallelism/03_consistent_tensor.md)) for more details.
```

tensor of tensor 冗余了

Reply:
已修改